lib/filter.js

```diff
@@ -117,6 +115,8 @@ filter.parseAndValidate = request => {
   for (const key in request.params.filter) {
     filterElement = request.params.filter[key]
 
+    if (typeof filterElement === 'string') request.params.filter[key] = filterElement = filterElement.split(FILTER_SEPERATOR)
```

Comment:
This is a pragmatic solution, but I'm not too thrilled about mutating `request.params.filter[key]`. I'm going to have a look at the internal routing and check if we can used the processed filter there also.